e2e/utils/visual-regression.js

```diff
@@ -292,5 +292,5 @@ module.exports.takeScreenShotOfElementWithCrop = (selector, options) => {
   });
   return browser.waitUntil(() => {
     return promise;
-  }, 1e4);
+  }, 1e5);
```

Comment:
1min40? i thought 1e4 is just too much:D

can we keep then 1e4 and increase to 20000 if necessary?

Reply:
No, not necessary, I think it does not matter.